nipype/interfaces/base/core.py

```diff
@@ -1267,6 +1267,35 @@ def _format_arg(self, name, spec, value):
         return super(SEMLikeCommandLine, self)._format_arg(name, spec, value)
 
 
+class LibraryBaseInterface(BaseInterface):
+    _pkg = None
+    imports = ()
+
+    def __init__(self, check_import=True, *args, **kwargs):
+        super(LibraryBaseInterface, self).__init__(*args, **kwargs)
+        if check_import:
+            import importlib
+            failed_imports = []
+            for pkg in (self._pkg,) + tuple(self.imports):
+                try:
+                    importlib.import_module(pkg)
+                except ImportError:
+                    failed_imports.append(pkg)
+            if failed_imports:
+                iflogger.warn('Unable to import %s; %s interface may fail to '
```

Comment:
is this going to generate a lot of warnings when we create docs? or `make specs`? i.e. things that simply create an instance of an interface:

`InterfaceX()`

Reply:
Yeah, potentially. I guess we could set a class-level flag to only display once per class? Or do you have another preferred approach?

Reply:
one option would be to remove this if. let's merge it and see how annoying it gets.